llama_index/vector_stores/milvus.py

```diff
@@ -270,9 +275,19 @@ def query(self, query: VectorStoreQuery, **kwargs: Any) -> VectorStoreQueryResul
 
         # Parse the results
         for hit in res[0]:
-            node = metadata_dict_to_node(
-                {"_node_content": hit["entity"].get("_node_content", None)}
-            )
+            if not self.text_key:
+                node = metadata_dict_to_node(
+                    {"_node_content": hit["entity"].get("_node_content", None)}
+                )
+            else:
+                try:
+                    text = hit["entity"].get(self.text_key)
+                except Exception as e:
+                    print("The passed in text_key value does not exist in the retrieved entity.")
+                    print(e)
```

Comment:
Should this just raise a `ValueError()` instead? There's no text to make the node 👀

Reply:
makes sense, i'm not familiar with the natives yet, accepted